lib/widgets/draggable_sheet_widget.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:Medito/constants/constants.dart';
+import 'package:flutter/material.dart';
+
+class DraggableSheetWidget extends StatelessWidget {
+  const DraggableSheetWidget({super.key, required this.child});
+  final Widget Function(ScrollController scrollController) child;
+
+  @override
+  Widget build(BuildContext context) {
+    return DraggableScrollableSheet(
+      initialChildSize: 0.5,
+      minChildSize: 0.4,
+      maxChildSize: 1,
+      expand: false,
+      builder: (
+        BuildContext context,
+        ScrollController scrollController,
+      ) {
+        return Container(
+          decoration: BoxDecoration(
+            color: ColorConstants.onyx,
+            borderRadius: BorderRadius.only(
+              topLeft: Radius.circular(12),
```

Comment:
Can we make a constants class for these? 
Maybe like MeditoCircularRadiusLarge, MeditoCircularRadiusMedium, MeditoCircularRadiusSmall, something like that?

Reply:
Great idea, yes we can do that as well

Reply:
Hey @michaelcspeed, I just checked the problem is we don't have a constant radius in the app. We use a different radius in multiple locations.

Reply:
@scouech Can you check how many there should be please?